.github/workflows/shopify-cli.yml

```diff
@@ -220,3 +222,5 @@ jobs:
       - name: Acceptance tests
         if: ${{ matrix.node == '18.7.0' }}
         run: yarn nx run features:test
+      - name: Golden tests
```

Comment:
What about building it upon the existing Cucumber setup?

```ghekin
Scenario: No runtime errors happen when loading the modules graph
  Then: I run help for every command available
```
In my experience working with bash scripts is usually friction when contributing.